Project-18-Food-order-app/src/components/UI/Button/index.jsx

```diff
@@ -0,0 +1,21 @@
+import PropTypes from "prop-types";
+import style from "./button.module.scss";
+
+function Button({ children, textOnly, className, ...prop }) {
+  let cssClasses = textOnly ? style["text-button"] : style["button"];
+  cssClasses += " " + className;
```

Comment:
@sknepali You're concatenating class names using string concatenation. This is fine, but you can use the classnames library to make this cleaner and more readable.
```
const cssClasses = classNames({
    [style.textButton]: textOnly,
    [style.button]: !textOnly,
    [className]: className,
 });
  ```